src/popup/components/buttons/BtnHelp.vue

```diff
@@ -23,12 +23,18 @@ export default defineComponent({
     icon: { type: String, default: '' },
     option: { type: Object, default: null },
     small: Boolean,
+    isCustomEmit: Boolean,
   },
-  setup(props) {
+  setup(props, { emit }) {
     const { openModal } = useModals();
 
-    async function showHelpModal() {
-      return openModal(MODAL_HELP, {
+    function showHelpModal() {
+      if (props.isCustomEmit) {
+        emit('help');
+        return;
+      }
```

Comment:
I see the problem you want to solve here. But maybe approaching it in a little bit different way would be better. What I suggest is:
1. Remove the `isCustomEmit` prop.
2. Fire `emit('help')` always (without wrapping it with `if` statement`
3. Open the `MODAL_HELP` only when there's a `msg` and/or `title` props.

What do you think?